examples/simulated-light/light.ts

```diff
@@ -27,19 +12,19 @@ import {
   STATES as MEDIAPLAYER_STATES,
   DEVICECLASSES as MEDIAPLAYER_DEVICECLASSES
 } from "../../lib/entities/media_player.js";
-
 import { COMMANDS as BUTTONCOMMANDS } from "../../lib/entities/button.js";
 import { STATUS_CODES } from "http";
 import { DEVICE_STATES, EVENTS as API_EVENTS } from "../../lib/api_definitions.js";
 import { CommandHandler } from "../../lib/entities/entity.js";
-
 import {
   COMMANDS as LIGHT_COMMANDS,
   STATES as LIGHT_STATES,
   ATTRIBUTES as LIGHT_ATTRIBUTES,
   FEATURES as LIGHT_FEATURES
 } from "../../lib/entities/light.js";
 
+uc.init("dist/examples/simulated-light/light-driver.json");
```

Comment:
See previous PR review: one needs to change into the example directory first.
This is simply: `uc.init("light-driver.json");`